src/client/netsblox.js

```diff
@@ -332,22 +332,15 @@ NetsBloxMorph.prototype.openIn = function (world) {
                         },
                         function () {nop(); }, // yield (bug in Chrome)
                         function () {
-                            SnapCloud.callService(
-                                'getProject',
-                                function (response) {
-                                    SnapActions.openProject(response[0].SourceCode, dict.Public)
-                                        .accept(function() {
-                                            msg.destroy();
-                                            applyFlags(dict);
```

Comment:
don't we need this part to open projects in appMode

Reply:
I just checked it breaks that functionality

Reply:
Good catch